src/util/promisified/fs.js

```diff
@@ -0,0 +1,11 @@
+const { promisify } = require('util');
```

Comment:
All of these can just be replaced with `pify(moduleRef)` using https://github.com/sindresorhus/pify

Reply:
Per https://github.com/sindresorhus/pify/issues/41 I think we should stick with native `util.promisify`, since it works with the `util.promisify.custom` symbol. That gives us a standard way we can normalize Node APIs with multi-argument callbacks, and the [Node documentation reflects this](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback). I like `pify`, but its [behavior with things like `child_process.exec`](https://github.com/sindresorhus/pify#multiargs) is now nonstandard, so I think we should stay with the native here.

Reply:
Cool! TIL